troposphere/static/js/components/images/detail/versions/Version.react.js

```diff
@@ -120,41 +163,55 @@ export default React.createClass({
                         type={type}
                     />
                 }
-                description={
-                    <div
-                        style={ style.content }
-                    >
-                        <div style={ style.description }
-                            dangerouslySetInnerHTML={{ __html: changeLogHTML }}
-                        />
-                        <div style={ style.availability }
-                        >
-                            { this.renderAvailability() }
-                        </div>
-                    </div>
-                }
+                summary={ this.renderDetail() }
+                detail={ this.renderDetail() }
```

Comment:
Yes, it is intentional. I can comment. In this case, the only difference between the detail and summary is the UUID being shown or not, which is handled by toggling a prop on the availability component.